torch/distributed/_tensor/op_schema.py

```diff
@@ -161,6 +161,14 @@ def output_ndim(self):
     def output_shape(self):
         return self.strategies[0].output_spec.shape
 
+    @property
+    def ndim(self):
```

Comment:
why we need to add these two properties, instead of just directly call `output_ndim` and `output_shape`?

Reply:
This is to support the existing test in `text_view_ops.py`. As discussed offline, will send a follow up PR to deprecate `output_ndim` and `output_shape`.